data-tool/flows/config.py

```diff
@@ -161,7 +161,7 @@ class _Config():  # pylint: disable=too-few-public-methods
     DB_NAME_COLIN_ORACLE = os.getenv('DATABASE_NAME_COLIN_ORACLE', '')
     DB_HOST_COLIN_ORACLE = os.getenv('DATABASE_HOST_COLIN_ORACLE', '')
     DB_PORT_COLIN_ORACLE = os.getenv('DATABASE_PORT_COLIN_ORACLE', '1521')
-    SQLALCHEMY_DATABASE_URI_COLIN_ORACLE = 'oracle+oracledb://{user}:{password}@{host}:{port}/{name}'.format(
+    SQLALCHEMY_DATABASE_URI_COLIN_ORACLE = 'oracle+oracledb://{user}:{password}@{host}:{port}/?service_name={name}'.format(
```

Comment:
i can only get it working if i use this format for the connection string.  otherwise I get a TNS listener issue

Reply:
@chenhongjing does this still work for you if i use this connection string format?